content/en/blog/2024/k8s-upstream-training-japan-spotlight/k8s-upstream-training-japan-spotlight.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: blog
+title: "Spotlight on Kubernetes Upstream Training in Japan"
+slug: k8s-upstream-training-japan-spotlight
+date: 2024-10-28
+author: >
+  [Junya Okabe](https://github.com/Okabe-Junya) (University of Tsukuba)
+  Organizing team of Kubernetes Upstream Training in Japan
```

Comment:
I think this will make more sense
```suggestion
  [Junya Okabe](https://github.com/Okabe-Junya) (University of Tsukuba) / 
  Organizing team of Kubernetes Upstream Training in Japan
```